src/auth/constants/authConstants.ts

```diff
@@ -0,0 +1,7 @@
+export default {
+  URLS: {
+    REGISTRATION_FORM:
+      'http://jassari.munstadi.fi/files/2019/06/Jasenkorttikaavake_2019_suomi.pdf',
```

Comment:
A form in Finnish language is provided for all language versions. I would imagine that we have Swedish and English version available--we should consider making those available.

Reply:
Added links for english and swedish registration forms. Somehow didn't realize I could change language at `http://jassari.munstadi.fi` to get links for those. :sweat_smile: